modules/nf-core/umitools/dedup/main.nf

```diff
@@ -1,11 +1,11 @@
 process UMITOOLS_DEDUP {
     tag "$meta.id"
-    label "process_medium"
+    label "process_single"
```

Comment:
Hmm, is this sufficient RAM?

Reply:
Good spot! @MatthiasZepper downgraded as explaind here https://github.com/nf-core/modules/pull/2971#issue-1611889023 . We can always bump in the next release if it breaks for users or they can update the resources via a config.

Reply:
Hard to say how the new version behaves, haven't tested the pipeline with large datasets. If you like, I could launch a test on our HPC cluster?

PS: Great that you pointed out that PR, reading through my own commend reminded me, that UMI-tools extract now also supports variable separators. Therefore, I suggest we still add this to UMITOOLS_EXTRACT in the `modules.config` (analogous to UMITOOLS_DEDUP):

`params.umitools_umi_separator   ? "--umi-separator='${params.umitools_umi_separator}'" : '',`

Reply:
Sorry, can't test: `Nextflow version 21.10.6 does not match workflow required version: >=22.10.1 ` and we don't have that running yet on our HPC. We have the version in the module system, but it always tries to download some additional plugins which fails due to a lack of internet connection. I suppose I have to copy the `NXF_HOME` from the other cluster over to offline one, but 2h is enough for Sunday.

Reply:
So I have now tested the release candidate and indeed the new process label causes issues. 

`umi_tools extract` requires very little memory, but is slow, so the 4h time limit is ambitious. For most samples, the second task attempt with longer runtimes was required.

`umi_tools dedup`was always killed with a 137 status, so indeed requested way more memory than allowed. :-(

I think, we need to address this somehow. It is a bit unfortunate that nf-core has those combo process labels for CPU, memory and runtime instead of atomic separate ones. 

![umitools-extract](https://user-images.githubusercontent.com/6963520/228509875-6723c633-a39f-46f3-a10a-c112e37b21e0.png)
![umitools-runtime](https://user-images.githubusercontent.com/6963520/228509880-fddb0e0e-a1ae-4e64-a257-ac37b1ab9c92.png)

Reply:
I have [patched the modules and hope it helps for now](https://github.com/nf-core/rnaseq/pull/983). Ultimately, we should rethink the process label in the modules repo, unfortunately. :-(